### PR TITLE
Update prediction form styles

### PR DESCRIPTION
**Pull request by ehunter**
I changed the layout of this panel quite a bit from the mockups. The multiple steps in the original design of the form seemed unnecessary given the amount of space we have in this panel now that the description is gone.

This is only the style updates, nothing is hooked up yet. Interested to hear what you all think…

### CODE REVIEW

**Comment by mikec** [comment]
@ehunter looks great!! 👍

File: app/components/MakePredictionForm.js [hunk 1]
```diff
@@ -18,67 +21,112 @@ const MakePredictionForm = createReduxForm(({
...
-    <StyledField
-      name="noShortAmount"
-      component="input"
-      type="text"
-      placeholder="NO-SHORT Amount"
-    />
-    <br /><br />
+const StyledYesBadge = styled(Badge) `
+  background-color: #80AEDC;
+  color: white !important;
```
Comment (jpkcambridge) [inline app/components/MakePredictionForm.js]: Do we need the `!important` for this

Comment (ehunter) [reply app/components/MakePredictionForm.js]: @jpkcambridge good catch, don't need the !important here. I can put up a patch which removes this

Comment (jpkcambridge) [reply app/components/MakePredictionForm.js]: @ehunter Don't worry about the patch.  I can take them out while I'm hooking this up.

**Pull request merged**